src/Cache.php

```diff
@@ -23,10 +26,10 @@ public function get($key, $default = null);
      *
      * @param String $key The key/index for the cache entry
      * @param mixed $value The item to store in the cache
-     * @param int $ttl The time to live (or expiry) of the cached item. Not all caches honor the TTL.
+     * @param null|int $ttl The time to live of the cached item. Not all caches honor the TTL.
```

Comment:
Should this typehint DateInterval too?

```suggestion
     * @param null|int|DateInterval $ttl The time to live of the cached item. Not all caches honor the TTL.
```